src/frontend/src/eth/components/send/SendTokenWizard.svelte

```diff
@@ -248,6 +250,8 @@
 				{/if}
 			</svelte:fragment>
 		</SendForm>
+	{:else if currentStep?.name === WizardStepsSend.QR_CODE_SCAN}
+		<QRCodeScan expectedToken={$sendToken} bind:destination bind:amount {decodeQrCode} />
```

Comment:
Actually, is it correct to use `bind:` here?  Shouldn't it be just passing the props `{destination} {amount}`?

Reply:
I think so, since the QRCodeScan must change those amounts. Otherwise, the amount and destination will not change.

Reply:
Got it!